pyproject.toml

```diff
@@ -40,14 +39,15 @@ line-length = 120
 target-version = ['py35','py36','py37', 'py38']
 
 [tool.poetry.dependencies]
-python = ">=3.5"
+python = ">=3.6"
 jinja2 = "^2.10"
 gitpython = ">=2.1"
 docopt = "^0.6.2"
 Click = "^7.0"
 
 [tool.poetry.dev-dependencies]
 pytest = "^4.6"
+black = {version = "*", allow-prereleases = true}
```

Comment:
Why prereleases?

Reply:
> 
> 
> Why prereleases?

because
```shell
$ poetry add --dev  black="*"                                                                                                   21:50:41
Creating virtualenv auto-changelog-34NNEQCw-py3.8 in /home/stdtom/.cache/pypoetry/virtualenvs

Updating dependencies
Resolving dependencies... (0.0s)

[SolverProblemError]
Because auto-changelog depends on black (*) which doesn't match any versions, version solving failed.
```

Reply:
Now I see, there is no production release yet. All of them are betas.